src/executables/util_service.c

```diff
@@ -93,7 +93,8 @@ typedef enum
   SC_APPLYLOGDB,
   GET_SHARID,
   TEST,
-  REPLICATION
+  REPLICATION,
+  SC_RKCHECK
```

Comment:
us_Command_map[]의 RKCHECK를 SC_RKCHECK로 변경 또는 SC_RKCHECK를 RKCHECK로 변경; SC_ 로 특별한 구분을 하는 것 같지 않아 다음과 같이 제안은 드립니다.
```suggestion
  RKCHECK
```

Reply:
@hornetmj 반영했습니다.